server/routes/requests.js

```diff
@@ -74,4 +74,61 @@ router.post('/addRequest', async (req, res, next) => {
     }
 });
 
+router.patch('/updateStatus/:id/:timeOffset', async (req,res, next) => {
```

Comment:
Note: this will have to be renamed when I have scheduling set up, because mentees will be able to update status by canceling an appointment